mastering/custom_cache.rst

```diff
@@ -88,3 +88,19 @@ It is possible to disable the locking mechanism in ``conan.conf``:
 
     [general]
     cache_no_locks = True
+
+System Requirements
```

Comment:
I really think this whole explanation should not be here but in reference/commands/remove. If you want to add a note here with a reference link and a brief explanation it would be fine.

Reply:
@danimtb I've moved the command line, but the introduction I kept here, please, review again and comment wdyt.